### DISCUSSION TITLE
Is it possible to disable title edit for singleton modules?

### DISCUSSION THREAD

**Discussion by stuartmcgill** [area17/twill#2347] [Q&A] [answered]
Is it possible to turn off the ability to edit the title for [singleton modules](https://twillcms.com/docs/modules/singletons.html)?

(I've added some back-end validation to the generated `Request` object to prevent the user doing this, but it would be nice to remove the option from the front-end too).

![Screenshot from 2023-10-18 18-36-30](https://github.com/area17/twill/assets/30128592/1c23a40b-23b5-4b80-bdc3-2cb5cbef060b)

**Comment by kallefrombosnia** [comment, marked as answer]
For the title, you can use 'editableTitle' => false in the controller's formData method.

**Reply by stuartmcgill** [reply]
Thank you, it works perfectly :)

(Out of interest, are the possible `formData` settings documented anywhere? - I saw the `editableTitle` check in the Title Editor vue class, but couldn't figure out from the code how to set it).

**Reply by kallefrombosnia** [reply]
I think there is no all defined formData properties which are used in the components, but you can find plenty of examples for specific usage via search on docs.
